README.md

```diff
@@ -70,10 +70,7 @@ calculatorService.Register("subtract", func(req server.Request, res *server.Resp
     // subtraction logic here...
 })
 
-calculatorService.Start()
```

Comment:
@alexandrevicenzi this method was replaced by `ListenAndServe` that is sync. You'll need to use `go service.ListenAndServe()` to continue the execution.